test/e2e/common.go

```diff
@@ -77,13 +76,15 @@ type linuxInstallerTestSuite struct {
 
 // SetupSuite is called at suite initialisation, once before all tests
 func (s *linuxInstallerTestSuite) SetupSuite() {
+	s.Suite.SetupSuite()
 	t := s.T()
 	if flavor == "" {
 		t.Log("setting default agent flavor")
 		flavor = defaultAgentFlavor
 	}
 	s.baseName = baseNameByFlavor[flavor]
 	s.configFile = configFileByFlavor[flavor]
+	s.Env().VM.CopyFolder(scriptPath, "scripts")
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
To allow running this from local tests, you could check an env var or flag and copy the scripts folder only on the CI. Or when local, download the scripts using the curl command and save it locally under a "scripts" folder, so that this code works for both local and CI

Reply:
To trigger locally you mean using the
```
cd test/e2e && gotestsum --format standard-verbose --junitfile "junit-${CI_JOB_NAME}.xml" -- -timeout 0s . -v --flavor ${FLAVOR} --platform ${PLATFORM} --scriptPath ${SCRIPT_PATH}  ${EXTRA_PARAMS}
```
command? I will test, it would just require to copy the generated scripts in a dedicated folder, correct?